Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1

```diff
@@ -142,6 +146,7 @@
 
     if (!$SkipPackageValidation)
     {
+        $status.value = "NewApp-TestingApplicationPackage"
         $packageValidationSuccess = (Test-ServiceFabricApplicationPackage $AppPkgPathToUse)
```

Comment:
Let's remove redundant Test-. Keep it only in deploy.ps1

Reply:
Please file an issue for that - otherwise we will miss it.

Reply:
#7492